src/main/scala/cromwell/engine/package.scala

```diff
@@ -175,7 +175,7 @@ package object engine {
 
   case class SymbolStoreKey(scope: String, name: String, index: Option[Int], input: Boolean)
 
-  case class SymbolStoreEntry(key: SymbolStoreKey, wdlType: WdlType, wdlValue: Option[WdlValue]) {
+  case class SymbolStoreEntry(key: SymbolStoreKey, wdlType: WdlType, wdlValue: Option[WdlValue], isWorkflowOutput: Boolean) {
```

Comment:
Passing around a boolean flag like this starts sending off the "ADT" signal in my brain